swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -127,103 +187,77 @@ service /covid/status on new http:Listener(9000) {
 In this code:
 
 - Unlike normal functions, resource methods can have accessors. In this case, the accessor is set to `get`, which means only HTTP `GET` requests could hit this resource. Ballerina automatically serializes Ballerina records as JSON and sends them over the wire. 
-- The default HTTP response status code for a resource method other than `post` is `200 OK`. For an HTTP `POST` resource, the default HTTP response status code is `201 Created`. 
+- The default HTTP response status code for a resource method other than `post` is `200 OK`. The default HTTP response status code for an HTTP `POST` resource is `201 Created`. 
 
 ### Create the second resource to add data
 
-To create the second resource of the first endpoint to add new COVID-19 data to the dataset by ISO code, add the code below to the API template file (i.e., `service.bal`).
+Create the second resource of the first endpoint to add new COVID-19 data to the dataset by ISO code, using the Ballerina HTTP API Designer in VS Code, as shown below.
+
+>**Note:** Define an HTTP resource that allows the `POST` operation on the resource path `countries` and accepts a `CovidEntry[]` type payload named `covidEntries`. Use `CovidEntry[]` and `IsoCodeConflict` as the response types.
```

Comment:
```suggestion
>**Note:** Define an HTTP resource that allows the `POST` operation on the resource path `countries` and a `CovidEntry[]`-typed `covidEntries` parameter corresponding to the payload. Use `CovidEntry[]` and `IsoCodeConflict` as the response types.
```

Reply:
Since `IsoCodeConflict` is only generated here, we can't directly say use it as a response type, IMO. We need to mention generating it.

Reply:
Updated the note as follows.

>**Note:** Define an HTTP resource that allows the `POST` operation on the resource path `countries` and accepts a `CovidEntry[]` type payload named `covidEntries`, create a `201 Created` response with the `CovidEntry[]` as the body type, and also create a `409 Conflict` response with `string` as the body type by generating a `IsoCodeConflict` record type of the `http:Conflict` status code response.